+ source ../common_lib/fsio_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
++ export 'FS_TYPE=ext3 ext4 xfs'
++ FS_TYPE='ext3 ext4 xfs'
++ export LANG=en_US.UTF-8
++ LANG=en_US.UTF-8
++ REMOVE_VG
++ vggroup=
++ [[ '' != '' ]]
+++ df -T
+++ grep test_vggroup
+++ awk '{print $7}'
++ mounted=
++ [[ '' != '' ]]
++ lsblk
++ grep test_vggroup
++ [[ 1 -eq 0 ]]
++ rm -rf '/tmp/point*'
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the database config.'
+ message='Start to prepare the database config.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the database config.'
Tue Aug 29 10:15:01 2023 - INFO  - Start to prepare the database config.
++ date +%Y%m%d%H%M%S
+ cur_date=20230829101501
++ CREATE_VG
+++ date +%Y%m%d%H%M%S
++ cur_date=20230829101501
+++ lsblk
+++ grep disk
+++ awk '{print $1}'
+++ tail -n 1
++ free_disk=vda
++ disk_name=/dev/vda
++ pvcreate /dev/vda -f
++ [[ 127 -eq 0 ]]
+++ pvcreate /dev/vda
+++ cut -d '"' -f 4
+++ grep test_vggroup
++ vggroup=
++ printf
printf: usage: printf [-v var] format [arguments]
+ vggroup=
+ lv=test_lv20230829101501
+ lvcreate -n test_lv20230829101501 -L 2G -y
oe_test_FSIO_create_ext3.sh: line 26: lvcreate: command not found
+ LOG_INFO 'Finish to prepare the database config.'
+ message='Finish to prepare the database config.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish to prepare the database config.'
Tue Aug 29 10:15:01 2023 - INFO  - Finish to prepare the database config.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue Aug 29 10:15:01 2023 - INFO  - Start to run test.
+ mkfs.ext3 /dev//test_lv20230829101501
mke2fs 1.47.0 (5-Feb-2023)
The file /dev//test_lv20230829101501 does not exist and no size was specified.
+ CHECK_RESULT 1 0 0 'mkfs.ext3 failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='mkfs.ext3 failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'mkfs.ext3 failed'
+ LOG_ERROR 'mkfs.ext3 failed'
+ message='mkfs.ext3 failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'mkfs.ext3 failed'
Tue Aug 29 10:15:01 2023 - ERROR - mkfs.ext3 failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_FSIO_create_ext3.sh line 33'
+ message='oe_test_FSIO_create_ext3.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_FSIO_create_ext3.sh line 33'
Tue Aug 29 10:15:01 2023 - ERROR - oe_test_FSIO_create_ext3.sh line 33
+ return 0
+ blkid
+ grep /dev/mapper/-test_lv20230829101501
+ grep 'TYPE="ext3"'
+ CHECK_RESULT 1 0 0 'Check lv infos failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lv infos failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lv infos failed'
+ LOG_ERROR 'Check lv infos failed'
+ message='Check lv infos failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lv infos failed'
Tue Aug 29 10:15:02 2023 - ERROR - Check lv infos failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_FSIO_create_ext3.sh line 35'
+ message='oe_test_FSIO_create_ext3.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_FSIO_create_ext3.sh line 35'
Tue Aug 29 10:15:02 2023 - ERROR - oe_test_FSIO_create_ext3.sh line 35
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue Aug 29 10:15:02 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Aug 29 10:15:02 2023 - ERROR - failed to execute the case.
+ exit 2
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_FSIO_create_ext3.sh post_test'
+ message='oe_test_FSIO_create_ext3.sh post_test'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'oe_test_FSIO_create_ext3.sh post_test'
Tue Aug 29 10:15:02 2023 - INFO  - oe_test_FSIO_create_ext3.sh post_test
